resources/generate-money-factory.php

```diff
@@ -49,15 +48,21 @@ public static function __callStatic(string $method, array $arguments): Money
 
     $methodBuffer = '';
 
-    $currencies = iterator_to_array(new Currencies\AggregateCurrencies([
+    $iterator = new Currencies\AggregateCurrencies([
         new Currencies\ISOCurrencies(),
         new Currencies\BitcoinCurrencies(),
-    ]));
+        new Currencies\CryptoCurrencies(),
+    ]);
 
+    $currencies = array_unique([...$iterator]);
     usort($currencies, static fn (Currency $a, Currency $b): int => strcmp($a->getCode(), $b->getCode()));
 
     /** @var Currency[] $currencies */
     foreach ($currencies as $currency) {
+        if (is_numeric($currency->getCode()[0])) {
+            // skip currencies that start with a number, as php function has to start with a letter or underscore
+            continue;
+        }
```

Comment:
There is currently only one crypto currency in resources starting with an integer, 1INCH. It would be good to include it. Perhaps I can rename it to ONEINCH.

Reply:
Codes starting a number are now transformed to start with the number expressed as a word.